src/lib/ThumbnailsSidebar.js

```diff
@@ -81,15 +84,38 @@ class ThumbnailsSidebar {
             const { bpPageNum: pageNumStr } = thumbnailEl.dataset;
             const pageNum = parseInt(pageNumStr, 10);
 
-            if (this.onClickHandler) {
-                this.onClickHandler(pageNum);
+            if (this.onThumbnailSelect) {
+                this.onThumbnailSelect(pageNum);
             }
         }
 
+        this.anchorEl.focus();
```

Comment:
This seems risky. We've seen complaints from customers about Preview autofocusing when it shouldn't.

Reply:
This only fires in the clickhandler of the thumbnail button, so you know that the user has focused here intentionally. We need to move focus to the container so that it's not lost when the button element is deleted by the virtual scroller.